ui/components/component-library/file-uploader/file-uploader.tsx

```diff
@@ -0,0 +1,299 @@
+import React, { useEffect, useState, useCallback } from 'react';
+import FileInput from 'react-simple-file-input';
+import {
+  Box,
+  BoxAlignItems,
+  BoxBackgroundColor,
+  BoxFlexDirection,
+  ButtonIcon,
+  ButtonIconSize,
+  Icon,
+  IconColor,
+  IconName,
+  IconSize,
+  Text,
+  TextColor,
+  TextVariant,
+  FontWeight,
+  twMerge,
+} from '@metamask/design-system-react';
```

Comment:
Using all new design system react compoennts

Reply:
Noted on this.. We will use design-system-react for new components and changes. Thank you for updating them for me.